website/docs/advanced/proxy/overview.md

```diff
@@ -0,0 +1,1258 @@
+
+## Available Options
+
+You can specify options for the Proxy either via a YAML file or with environment variables.
+
+<Tabs groupId="yaml-env">
+<TabItem value="yaml" label="YAML" default>
+
+<details open>
+  <summary><strong>Docker</strong></summary>
+
+When running the Proxy in docker, you can mount the YAML file as a volume.
+```shell
+docker run -d --name configcat-proxy \ 
+  -p 8050:8050 -p 8051:8051 -p 50051:50051 \
+  // highlight-next-line
+  -v <path-to-file>/options.yml:/cnf/options.yml \
+  configcat/proxy -c /cnf/options.yml
+```
+
+</details>
+
+<details>
+  <summary><strong>Standalone executable</strong></summary>
+
+Running the Proxy as a standalone executable, you can pass the YAML file via the `-c` argument.
+
+```shell
+configcat-proxy -c /path-to-file/options.yml
+```
+
+</details>
+
+</TabItem>
+<TabItem value="env-vars" label="Environment variables">
+
+<details open>
+  <summary><strong>Docker</strong></summary>
+
+When running the Proxy in docker, you can pass environment variables for the executing container.
+```shell
+docker run -d --name configcat-proxy \ 
+  -p 8050:8050 -p 8051:8051 -p 50051:50051 \
+  // highlight-next-line
+  -e CONFIGCAT_SDKS='{"<sdk-identifier>":"<your-sdk-key>"}' \
+  configcat/proxy
+```
+
+</details>
+
+<details>
+  <summary><strong>Standalone executable</strong></summary>
+
+Make sure the related environment variables are available for the Proxy's hosting process. For example:
+
+
+<Tabs>
+<TabItem value="shell" label="shell">
+
+```shell
+export CONFIGCAT_SDKS='{"<sdk-identifier>":"<your-sdk-key>"}'
+```
+
+</TabItem>
+<TabItem value="powershell" label="PowerShell">
+
+```powershell
+$Env:CONFIGCAT_SDKS='{"<sdk-identifier>":"<your-sdk-key>"}'
+```
+
+</TabItem>
+</Tabs>
+
+Then start the Proxy:
+
+```shell
+configcat-proxy
+```
+
+</details>
+
+</TabItem>
+</Tabs>
+
+The following sections will go through each available option in detail.
+
+### Logging
+
+The Proxy supports the following log levels: `debug`, `info`, `warn`, and `error`. The default is `warn`.
+
+You can specify the log level either globally or for each individual component. If you don't set a component's log level explicitly, it will inherit the global level's value.
+
+<Tabs groupId="yaml-env">
+<TabItem value="yaml" label="YAML" default>
+
+```yaml
+log:
+  level: "<error|warn|info|debug>"
+```
+
+</TabItem>
+<TabItem value="env-vars" label="Environment variables">
+
+```shell
+CONFIGCAT_LOG_LEVEL="<error|warn|info|debug>"
+```
+
+</TabItem>
+</Tabs>
+
+### SDK
+
+In order to make the Proxy work properly it must be set up with one or more <a target="blank" href="https://app.configcat.com/sdkkey">SDK keys</a>. It will instantiate one SDK instance for each SDK key.
+In case of environment variables the SDK keys can be specified in a **JSON key-value** format where the **key is the identifier** of a specific SDK and the **value is the actual SDK key**.
```

Comment:
```suggestion
In case of environment variables, the SDK keys can be specified in a **JSON key-value** format, where the **key is the identifier** of a specific SDK and the **value is the actual SDK key**.
```